packages/contracts/contracts/plugin/PluginRepo.sol

```diff
@@ -150,11 +153,29 @@ contract PluginRepo is
     //         contentURI: cid12
     // //  },
 
+    /// @inheritdoc IPluginRepo
+    function updateReleaseMetadata(
+        uint8 _release,
+        bytes calldata _releaseMetadata
```

Comment:
you can call this directly _metadata, function name already suggests what it is.

Reply:
amended